projects/documentation/content/guides/generating-components.md

```diff
@@ -13,50 +13,44 @@ The fastest way to get started with creating a new component is to generate it f
+> package-name
    > src/
        index.ts
-       SpectrumPattern.ts
-       spectrum-pattern.css
+       SpectrumPackageName.ts
+       package-name.css
+       package-name-overrides.css
+       spectrum-package-name.css
+       spectrum-package-name-overrides.css
        spectrum-config.js
    > test/
        > benchmark/
            basic-test.ts
-       spectrum-pattern.test.ts
+       package-name.test.ts
    > stories/
-       spectrum-pattern.stories.ts
-   sp-spectrum-pattern.ts
+       package-name.stories.ts
+   sp-package-name.ts
    README.md
    tsconfig.json
    package.json
 ```
 
-From here, peek into the `package.json` file and ensure the "devDependency" of `@spectrum-css/spectrumpattern` (replacing "spectrumpattern" with the pattern you're implementing, such as "badge" or "tooltip") listed there is of the most current version.
-
 Outside of your new package, you will need to manually add information about your new package to the `tsconfig-all.json` and `tsconfig-react-wrapper.json` files, as well as the `bundle` package:
 
-Open `tsconfig-all.json`, find "references", and add an entry for your package (`{ "path": "packages/spectrum-pattern" }`) alphabetically (replacing "spectrum-pattern" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-all.json` config is used to build types for the project in parallel with the JS build that is handled outside of `tsc`. This will ensure that the types for your new package are available throughout the library, include at demonstration and test time.
+Open `tsconfig-all.json`, find "references", and add an entry for your package (`{ "path": "packages/package-name" }`) alphabetically (replacing "package-name" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-all.json` config is used to build types for the project in parallel with the JS build that is handled outside of `tsc`. This will ensure that the types for your new package are available throughout the library, include at demonstration and test time.
 
-Open `tsconfig-react-wrapper.json`, find "references", and add an entry for your package (`{ "path": "react/spectrum-pattern" }`) alphabetically (replacing "spectrum-pattern" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-react-wrapper.json` config is used to build types for the `@swc-react` project, so that consumers can benefit of type definitions for SWC components even when using the React wrapper.
+Open `tsconfig-react-wrapper.json`, find "references", and add an entry for your package (`{ "path": "react/package-name" }`) alphabetically (replacing "package-name" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-react-wrapper.json` config is used to build types for the `@swc-react` project, so that consumers can benefit of type definitions for SWC components even when using the React wrapper.
```

Comment:
```suggestion
Open `tsconfig-react-wrapper.json`, find "references", and add an entry for your package (`{ "path": "react/package-name" }`), alphabetically (replacing "package-name" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-react-wrapper.json` config is used to build types for the `@swc-react` project, so that consumers can benefit from type definitions for SWC components even when using the React wrapper.
```